src/kernels/raw/session/rawSession.node.ts

```
@@ -105,9 +104,11 @@ export class RawSession implements ISessionWithSocket {
         this.isDisposing = true;
         if (!this.isDisposed) {
             this.exitHandler.dispose();
-            await this._kernel.shutdown();
+            await this._kernel
+                .shutdown()
+                .catch((ex) => traceWarning(`Failed to shutdown kernel, ${this.kernelConnectionMetadata.id}`, ex));
```

We can ignore the errors.